### PR TITLE
Move battery to its own file

### PR DESCRIPTION
**Pull request by Camerooooon**

### CODE REVIEW

**Comment by JakeRoggenbuck** [comment]
ay!

File: src/power/battery.rs [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        // Remove the \n char
+        cap_str.pop();
+
+        let charge = cap_str.parse::<i8>().unwrap();
+        self.capacity = charge;
+
+        Ok(())
+    }
+
+    // TODO: Move this to display.rs
```
Comment (JakeRoggenbuck) [inline src/power/battery.rs:92]: Would this make sense to be I display.rs?

Comment (JakeRoggenbuck) [reply src/power/battery.rs:92]: * in

Comment (Camerooooon) [reply src/power/battery.rs:92]: display.rs handles displaying things like power, turbo, cpus, etc. I think its fair for it to deal with printing battery info,

**Review by JakeRoggenbuck — APPROVED** [review]

**Pull request merged**